examples/todo-list/flutter-todo-list/lib/screens/signin.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:todosupabase/constant.dart';
+import 'package:todosupabase/functions/auth.dart';
+import 'package:todosupabase/functions/crud.dart';
+
+class SignInPage extends StatefulWidget {
+  const SignInPage({Key? key}) : super(key: key);
+
+  @override
+  State<SignInPage> createState() => _SignInPageState();
+}
+
+class _SignInPageState extends State<SignInPage> {
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+  bool loading = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SizedBox(
+        height: double.infinity,
```

Comment:
Same as the previous one, but I think we can get rid of the `SizedBox` here.

Reply:
Yes here also we need the sizedbox as mentioned in the same in the above comment